test/testdata/packager/export_imported/pass.package-tree.exp

```diff
@@ -9,17 +9,12 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
     module <emptyTree>::<C A_Package$1>::<C B><<C <todo sym>>> < ()
       <emptyTree>::<C BClass> = <emptyTree>::<C <PackageRegistry>>::<C B_Package$1>::<C B>::<C BClass>
     end
-
-    class <emptyTree>::<C A_Package$1>::<C A>::<C <Magic>><<C <todo sym>>> < ()
-    end
   end
 
   module <emptyTree>::<C <PackageTests>><<C <todo sym>>> < ()
     module <emptyTree>::<C A_Package$1>::<C B><<C <todo sym>>> < ()
-      <emptyTree>::<C BClass> = <emptyTree>::<C <PackageRegistry>>::<C B_Package$1>::<C B>::<C BClass>
+      <emptyTree>::<C BClass> = <emptyTree>::<C <PackageRegistry>>::<C A_Package$1>::<C B>::<C BClass>
```

Comment:
This is a little bit odd looking as the name comes from `A_Package` now, but I think the explanation is that it's using the re-exported version of the name instead of the original from `B_Package`?

Reply:
Yeah it's a weird case that I think we may want to ban in the future, but the names do end up all resolving.